### ISSUE TITLE
Cannot find Acer Chromebook 714 (CB714) EKKO UEFI Firmware anywhere

### ISSUE THREAD

**Issue by gotnt** [MrChromebox/firmware#672]
Hi, MrChromebox, the Acer Chromebook 714 (CB714) EKKO UEFI Firmware cannot be found while it shows up in the supported list, did I miss something? Thanks!

**Comment by MrChromebox** [comment]
what do you mean "cannot be found"?
What is the error?
what are you doing to cause the error?

this doesn't seem like a website issue at all

**Comment by gotnt** [comment]
Hi MrChromebox, sorry about not being specific enough. 
TLDR: I have a partially failed Acer CB714 EKKO that cannot be recovered to ChromeOS, the solution might be to use a ch341 to program the UEFI bios to install other os, but in here (https://github.com/MrChromebox/scripts/blob/master/sources.sh), the bios for this model couldn't be found, could you point me to the right direction? Thank you for your consideration

Long version:
I have an Acer CB714 EKKO that is in a boot loop and if I put it in recovery mode, it's giving me a message of ChromeOS is Missing or Damaged, please plug in a recovery usb etc. After following the guide of making a recovery media and put in a recovery usb, nothing happens appearently. The last resort came to mind was to flash your uefi bios and install windows or linux. After following your guide to unbrick a chromebooks, I'm stuck at downloading the correct bios. The bios for this model cannot be found in https://github.com/MrChromebox/scripts/blob/master/sources.sh and after some searching around with no result, I thought of seeking for help here. Thank you again for helping!

**Comment by MrChromebox** [comment]
per https://github.com/MrChromebox/scripts/blob/master/functions.sh#L664, EKKO uses the NAMI firmware image, so you download that one using the url in sources.sh
